src/passes/JSPI.cpp

```diff
@@ -49,6 +50,12 @@
 //      Wrap each export in the comma-separated list. Similar to jspi-imports,
 //      wildcards and separate files are supported.
 //
+//   --pass-arg=jspi-split-module
+//
+//      Enables integration with wasm-split. A JSPI'ed function named
+//      `__load_secondary_module` will be injected that is used by wasm-split to
+//      load a secondary module.
```

Comment:
And we can't have the wasm-split tool handle adding this function because wasm-split runs after Emscripten has already run the JSPI transformation on the module?

Reply:
Correct.